MiniApp/Classes/MiniApp.swift

```diff
@@ -71,6 +71,14 @@ public class MiniApp: NSObject {
         return []
     }
 
+    /// Gets the list of downloaded Mini apps info and associated custom permissions status
+    /// - Returns:Dictionary of MiniAppInfo and respective custom permissions info
+    public func listDownloadedWithCustomPermissions() -> [[MiniAppInfo: [MASDKCustomPermissionModel]]] {
```

Comment:
Could you also add this to the user guide?

Reply:
Done. Added in User guide and Changelog